src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          class="img_logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav_header">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav_header">
```

Reply:
Add empty lines between multiline sibling blocks of HTML

Reply:
fix it everywhere

Reply:
I didn’t understand this rule. Where can I read more about it, what is it for?